build/Build-FinOpsPsModule.ps1

```diff
@@ -0,0 +1,105 @@
+param
+(
+    [Parameter()]
+    [string]
+    $Version,
+
+    [Parameter()]
+    [ValidateSet('alpha', 'beta')]
+    [string]
+    $PrereleaseTag
+)
+
+$rootPath = (Get-Item -Path $PSScriptRoot).Parent.FullName
+$moduleName = 'FinOpsToolkit.psm1'
+$modulePath = Join-Path -Path $rootPath -ChildPath 'src\powershell'
+$privatePath = Join-Path -Path $modulePath -ChildPath 'private'
+$publicPath = Join-Path -Path $modulePath -ChildPath 'public'
+$releasePath = Join-Path -Path $modulePath -ChildPath 'FinOpsToolkit'
+
+if ($Version)
+{
+    if ($PrereleaseTag)
+    {
+        $versionString = "$Version-$PrereleaseTag"
+    }
+    else
+    {
+        $versionString = $Version
+    }
+
+    $releasePath = Join-Path -Path $releasePath -ChildPath $versionString
+}
+else
+{
+    $releasePath = Join-Path -Path $releasePath -ChildPath '0.0.1'
```

Comment:
Can we read the default version from a separate file? We probably don't need both code paths. We just need to set the default if a version isn't specified, right?